myia/ir/anf.py

```diff
@@ -50,6 +51,12 @@ def __init__(self) -> None:
         self.debug = NamedDebugInfo(self)
         self.flags = {}
         self.transforms: Dict[str, Union[Graph, Primitive]] = {}
+        self.vararg = False
+        self.kwarg = False
+        self.defaults = []
+        self.kwonly = 0
+        self.user_graph = None
+        self.sig = None
```

Comment:
These new attributes are not documented.

Reply:
I documented most of them, `user_graph` and `sig` are sort of for internal use so I put an underscore in front.